src/Ting/Repository/HydratorAggregator.php

```diff
@@ -47,6 +47,11 @@ class HydratorAggregator extends Hydrator
      */
     protected $callableFinalizeAggregate;
 
+    public function __construct()
```

Comment:
I think you can remove that constructor

Reply:
I removed the constructor and updated CHANGELOG